blocks/columns/columns.css

```diff
@@ -1,3 +1,7 @@
+.columns-wrapper {
+  margin-top: 1.5rem !important;
```

Comment:
can't we make the selector more specific?

Reply:
Fixed